cfme/tests/cli/test_appliance_console.py

```diff
@@ -40,6 +42,8 @@
     TZ('UTC', ('11',))
 ]
 RETURN = ''
+IPv6_REGEX = r'(IPv6 Address:\s*)(\w+:\w+:\w+:\w+:\w+:\w+:\w+:\w+)/'
```

Comment:
This is going to produce a flaky test, as this regex is insufficient to match the IPv6 spec.

Given the existing and planned support, I don't think this is worth including in the test suite.

Reply:
sure, will modify, thanks

Reply:
Looks like it was duplicated instead of replaced - even the updated expression is a bit too simple to catch the full spec. It may run within our labs, but its got a potential for not catching valid addresses.

Again given the scope of our coverage and use, and MIQ's planned support, I would advise against trying to exact match it here, or dropping the parametrization for IPv6.

You can just read from after `IPv6 Address: ` to the end of the line, and take what's there without regex matching the address exactly, if you want to keep the coverage but simplify the capture.

Reply:
Sorry, Mike, I have forgotten to remove old regex, could you please check the new one that is more mature than last one.